Restsplain.php

```diff
@@ -38,11 +38,13 @@ function get_default_docs_base() {
  * @return string
  */
 function get_docs_base() {
+	$subdir_install = (string) parse_url( home_url(), PHP_URL_PATH );
```

Comment:
I think we need to make sure there's no trailing slash here - there generally shouldn't be using `home_url()` without the path but it's a filterable function so best to be certain.

```suggestion
	$subdir_install = rtrim( (string) parse_url( home_url(), PHP_URL_PATH ), '/' );
```